Kernel/Arch/x86/ISRStubs.h

```diff
@@ -11,6 +11,54 @@
 #include <AK/Platform.h>
```

Comment:
Commit message nitpicks
`Even if PIC was disabled` => `Even if the PIC was disabled`
`so we need two handlers being registered for handling such case.` => `so we need to register two handlers for handling such cases.`
`we declare interrupt service routine offset 0x90 to 0x9f as` this is not correct anymore
`to a different interrupt handlers at` => `to different interrupt handlers at`
`at different location.` => `at different location.s`